command/commands/team.py

```diff
@@ -173,7 +181,8 @@ def handle(self, command, user_id):
         else:
             return self.get_help(), 200
 
-    def create_helper(self, param_list):
+    def create_helper(self,
+                      param_list: Dict[str, str]) -> ResponseTuple:
         """
```

Comment:
Some of the changes here may conflict with PR for finishing team commands, not sure if you wanna push these now and we could merge later or to remove the changes for now.